docs/components/modeler/forms/form-element-library/forms-element-library-number.md

```diff
@@ -14,12 +14,16 @@ A number field allowing the user to read and edit numeric data.
 - **Field description**: Description provided below the number field.
 - **Key**: Binds the field to a form variable, see [data binding docs](../configuration/forms-config-data-binding.md).
 - **Default value**: Provides a default value for the number field in case no input data exists for the given key.
+- **Decimal digits**: Defines the maximum number of digits after the decimal.
+- **Increment**: Defines the number of steps in which the number increments.
```

Comment:
I'm not sure about 'number of steps' here, but I'm also not sure what would be better.

Reply:
I don't really find a better word. Maybe @christinaausley can propose a better term.

Reply:
Maybe this?

```suggestion
- **Increment**: Defines the amount by which the field is incremented/decremented when using the control buttons.
```

Reply:
Unfortunately not, because it's not what this setting does. If you set the increment to 10, write 5 into the field and increment, you won't get 15, you'll get 10. 

Basically, it sets the increments between valid field values centred around 0.

So if your increment is 0.25, your valid values will be 0, 0.25, 0.5, 0.75, 1, ...

So maybe this should be something like, `defines the increment between valid field values`.

Reply:
Good point about the validation aspect. Adjusted it accordingly.